dietpi/dietpi-software

```diff
@@ -16100,11 +16113,6 @@ _EOF_
 		fi
 
 		# Interactive software post-configuration
```

Comment:
```suggestion
```

Reply:
The latest remoteit package differs from the old package connectd in its configuration files and how it works, so data cannot be migrated. 
In a few more words, they operate as separate packages, so each works simultaneously.
Users using the connectd package can install the latest remoteit package, configure their settings according to the connectd package, and then unpackage and uninstall connectd once that is done.
We have not removed the old package connectd from our support, and some users continue to use it. However, we want as many people as possible to use the latest packages. I have noticed that our package installed with DietPi is several years old so I would like put the latest remoteit package into the DietPi software list.

Reply:
We should then also add this info during/after install (if `dpkg-query -s 'connectd' &> /dev/null`) and to our docs.